templates/1_Segment_Image_Data.ipynb

```diff
@@ -97,7 +97,9 @@
     "\n",
     "Here we are using the example data located in `/data/example_dataset/input_data`. To modify this notebook to run using your own data, simply change the `base_dir` to point to your own sub-directory within the data folder, rather than `'example_dataset'`. \n",
     "\n",
-    "If you would like to test the features in Ark with an example dataset, run the cell below. It will download a dataset consisting of 11 FOVs with 22 channels. You may find more information about the example dataset in the [README](../README.md#example-dataset)."
+    "If you would like to test the features in Ark with an example dataset, run the cell below. It will download a dataset consisting of 11 FOVs with 22 channels. You may find more information about the example dataset in the [README](../README.md#example-dataset).\n",
+    "\n",
+    "* `overwrite_existing`: If set to `False`, it will not overwrite existing data in the `data/example_dataset`. Recommended setting it to `False` for Notebook 1, so it will always download the example_dataset data."
```

Comment:
Did you mean "Recommended setting it to `True` for Notebook 1, so it will always download the example_dataset data."?

Reply:
Yeah.